server/datastore/mysql/policies.go

```diff
@@ -431,3 +438,46 @@ func amountPoliciesDB(db sqlx.Queryer) (int, error) {
 	}
 	return amount, nil
 }
+
+// AsyncBatchInsertPolicyMembership inserts into the policy_membership_history
+// table the batch of policy membership results.
+func (ds *Datastore) AsyncBatchInsertPolicyMembership(ctx context.Context, batch []fleet.PolicyMembershipResult) error {
+	// NOTE: this is tested via the server/service/async package tests.
+
+	// TODO: INSERT IGNORE, to avoid failing if policy / host does not exist?
```

Comment:
Should we care about this ^ ? I.e. given that this is asynchronous, it's possible that the policy or host do not exist anymore once this insert gets executed.

Reply:
:+1: updated.